src/app/aria-selected.directive.ts

```diff
@@ -0,0 +1,29 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+// AriaSelectedMSPivotLinkDirective - An attribute directive that uses the ms-Pivot-link class selector to 
+// to apply the aria-selected attribute and value when the tab gains and loses focus.
+// https://dev.office.com/fabric-js/Components/Pivot/Pivot.html
+import { Directive, ElementRef, Renderer2, HostListener } from '@angular/core';
+@Directive({
+    selector: '.ms-Pivot-link'
+})
+export class AriaSelectedMSPivotLinkDirective {
+    constructor(private el: ElementRef, private renderer: Renderer2) {
+        this.el = el;
+        this.renderer = renderer;
+    }
+
+    // Set the aria-selected attribute to true when the tab (ms-pivot-link) gains focus.
+    @HostListener('focus')
```

Comment:
is this the right listener?  This will add "selected" to the narrator if he just hovers over the element, even if the element is not selected.

Reply:
If a user right-clicks with the mouse, it both gives focus and selects the element. I think this behavior is fine.
If a user tabs to the element, it only gives focus. They will still need press Enter to select the item. I think this is still acceptable. 

https://www.stefanjudis.com/blog/aria-selected-and-when-to-use-it/#the-actual-use-case-of-code-aria-selected-code- supports this scenario.